README.md

```diff
@@ -27,10 +29,19 @@ When the reader has completed this Code Pattern, they will understand how to:
 3. User input is processed and routed to the backend server, which is responsible for server side rendering of the views to be displayed on the browser. The backend server is written using express and uses express-react-views engine to render views written using React.
 4. The backend server sends user requests to the Watson Discovery Service. It acts as a proxy server, forwarding queries from the frontend to the Watson Discovery Service API while keeping sensitive API keys concealed from the user.
 
+### How does Watson Knowledge Studio work?
+
+The image below explains the process of how Watson Knowledge Studio works in light detail. For greater detail see Steps [4. Upload Type System](#4-upload-type-system) through [9. Deploy the machine learning model to NLU](#9-deploy-the-machine-learning-model-to-nlu).
```

Comment:
this comment doesn't make sense, update it after

Reply:
Will do, just a place holder for now